nbproject/project.properties

```diff
@@ -36,10 +36,11 @@ javac.classpath=\
 # Space-separated list of extra javac options
 javac.compilerargs=
 javac.deprecation=false
+javac.external.vm=false
 javac.processorpath=\
     ${javac.classpath}
-javac.source=1.5
-javac.target=1.5
+javac.source=1.7
+javac.target=1.7
```

Comment:
Not totally against updating the source level but not sure how important it is if we are not using any newer APIs or language features.